pkg/apb/types.go

```diff
@@ -45,6 +45,11 @@ type Spec struct {
 	Parameters []*ParameterDescriptor `json:"parameters"`
 }
 
+type Context struct {
+	Platform  string `json:"platform"`
```

Comment:
All I've seen so far is "kubernetes" and I haven't found the spec to see what else is possible.
```json
{
  "organization_guid": "996e99e2-4c7f-11e7-acf8-54ee755dba17",
  "plan_id": "4c10ff42-be89-420a-9bab-27a9bef9aed8",
  "service_id": "55c53a5d-65a6-4c27-88fc-e027410b1882",
  "space_guid": "996e99e2-4c7f-11e7-acf8-54ee755dba17",
  "accepts_incomplete": true,
  "context": {
    "platform": "kubernetes",
    "namespace": "hello-world"
  }
}
```